src/api.js

```diff
@@ -150,6 +163,11 @@ const endpoints = {
     const paginationParam = `requestedPage: {limit:${pageSize} offset:${(pageNumber - 1) * pageSize} sortBy:RELEVANCE}`;
     return `/v2/_zot/ext/search?query={GlobalSearch(${searchParam}, ${paginationParam}) {Images {RepoName Tag}}}`;
   },
+  imageSuggestionsWithPlatforms: ({ searchQuery = '""', pageNumber = 1, pageSize = 15 }) => {
```

Comment:
Is there a point for this endpoint? Can we not achieve this already with the 'globalSearch'?